Source/WebCore/page/scrolling/ScrollingStateNode.h

```diff
@@ -315,6 +315,8 @@ class ScrollingStateNode : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPt
     Vector<Ref<ScrollingStateNode>>& children() { return m_children; }
     const Vector<Ref<ScrollingStateNode>>& children() const { return m_children; }
     Vector<Ref<ScrollingStateNode>> takeChildren() { return std::exchange(m_children, { }); }
+    WEBCORE_EXPORT void setChildren(Vector<Ref<ScrollingStateNode>>&&);
+    void recurse(const Function<void(ScrollingStateNode&)>&);
```

Comment:
There exists `ScrollingTree::traverseScrollingTree()`. Maybe we should add `ScrollingStateTree:: traverse()` instead of this?